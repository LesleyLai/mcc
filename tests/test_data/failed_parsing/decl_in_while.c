int main(void) {
  while (1) int x = 0;
  return 0;
}
