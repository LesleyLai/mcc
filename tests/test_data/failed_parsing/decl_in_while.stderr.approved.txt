{{filename}}:2:13: Error: Expect valid expression
2 |   while (1) int x = 0;
  |             ^~~

